### PR TITLE
Feature/menu

### PR DESCRIPTION
**Pull request by arman-tdev**
implemented help and about menu.

### CODE REVIEW

**Comment by dileep-gadiraju** [comment]
@arman-tdev / @apurvabisht97/ @navin9584  ,  Please check if you can find better image for v1.0/frontend/SaralApp/src/assets/images/about.png. current one doesn't look great.

**Comment by apurvabisht97** [comment]
sure @dileep-gadiraju

File: v1.0/frontend/SaralApp/saralAppInfo.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "saral.release.version" : "v1.5.1",
+    "saral.info" : "Sunbird Saral is a OCR plus mobile application to physitize data from physical layout.",
+    "saral.documentation.link" : "https://saral.sunbird.org/",
+    "release.link" : "https://saral.sunbird.org/engage/releases"
```
Comment (apurvabisht97) [inline v1.0/frontend/SaralApp/saralAppInfo.json:5]: check this @dileep-gadiraju @arman-tdev

**Comment by apurvabisht97** [comment]
hi @dileep-gadiraju  i have tested above pr . this pr is working as expected .

**Pull request merged**